CIP-0015/CIP-0015.md

```diff
@@ -93,17 +125,33 @@ Here an example using CBOR diagnostic notation
 {
   61284: {
     1: h'8253C95609BC62C0443276FE2A1872B87CB11C06185FFDBB56C7CE8352EEF2A3',
-    2: h'345080C6DDFF7154B4ED4A622558AA0EAABD8CE7E2701C92B6858EA76DCECBCE'
+    2: h'AD4B948699193634A39DD56F779A2951A24779AD52AA7916F6912B8EC4702CEE',
+    3: h'00588E8E1D18CBA576A4D35758069FE94E53F638B6FAF7C07B8ABD2BC5C5CDEE47B60EDC7772855324C85033C638364214CBFC6627889F81C4',
+    4: 5479467
   },
-  61285: {
-    1: h'7D88F34D778B7A4C76AA53FF5D9506DC5B92D25575B43AF75D66DC05082A2BCFF44FCEDDAB15DBA0C23C56A09A15367A9803E24A388AAFB8498EF72190407B0D'
-  }
 }
 ```
```

Comment:
This would better be in a separate annex file which can be referenced / downloaded as-is.